cypress/support/commands.js

```diff
@@ -1,3 +1,12 @@
+Cypress.Commands.add('findByType', (type) => {
+    cy.get(`[type="${type}"]`);
+ });
+ 
+ 
+//  Cypress.Commands.add('findByPlaceholder', (placeholder) => {
```

Comment:
There is an unnecessary commented-out code block. If it's not needed, consider removing it to keep the code clean and maintainable 🧹